api/controllers/shabads.js

```diff
@@ -9,7 +9,7 @@ const allColumns = `v.ID, v.Gurmukhi, v.GurmukhiUni, v.Translations, v.PageNo AS
     v.SourceID as SourceID, s.ShabadID, v.FirstLetterStr, v.MainLetters, v.Visraam,
     v.FirstLetterEng, v.Transliterations, v.WriterID, w.WriterEnglish,
     w.WriterGurmukhi, w.WriterUnicode, v.RaagID, r.RaagGurmukhi,
-    r.RaagUnicode, r.RaagEnglish, r.RaagWithPage, r.StartID, r.EndID,
```

Comment:
hope no one using API was relying on those columns.

Reply:
if they were their use case is very very broken. Tarsem singh just told us this is from some very old version of STTM and has no use